docs/docs/guides/labs/dg/multiple-projects.md

```diff
@@ -17,42 +17,36 @@ If you're just getting started, we recommend [scaffolding a single project](/gui
 
 If you need to collaborate with multiple teams, or work with conflicting dependencies that require isolation from each other, you can scaffold a workspace directory that contains multiple projects, each with their own separate Python environment.
 
-A workspace directory contains a root `dg.toml` with workspace-level settings, and a `projects` directory with one or more projects.
-
-:::note
-
-A workspace does not define a Python environment by default. Instead, Python environments are defined per project.
-
-:::
+A workspace directory contains a root `dg.toml` with workspace-level settings, and a `projects` directory with one or more projects. It also contains a Python environment in a `deployments/local` folder that can be used for running `dg` commands locally against the workspace.
```

Comment:
At some point a simple architecture diagram showing which processes are in what environment would be good (e.g. webserver in deployment environment, code server in project environment)